Documentation/Deployment.md

```diff
@@ -3,9 +3,9 @@
 ## Template deployment via Safe Deploy process
 Once content is checked into master, the template content is packaged into an npm package, and the release pipeline will publish that npm package to our internal team packaging feed where other dependencies live. That npm package of templates will be picked up by the AppInsights extension's daily build, and deployed via the extension's daily release train.
 
-1. After a PR is completed to the `master` branch, an [official build is queued](https://github-private.visualstudio.com/microsoft/_build?definitionId=48) (you might not have access to this), which packages all the processed templates into a versioned NPM package
+1. An [official build is queued](https://github-private.visualstudio.com/microsoft/_build?definitionId=474) (you might not have access to this), on a scheduled daily basis which packages all the processed templates into a versioned NPM package
 
-2. After the official build completes, [a release pipeline](https://github-private.visualstudio.com/microsoft/_release?_a=releases&definitionId=1) (again, you might not have access to this), uploads the built NPM package to an Azure Devops package feed.
+2. After the official build completes, [a release pipeline](https://github-private.visualstudio.com/microsoft/_release?_a=releases&view=mine&definitionId=65) (again, you might not have access to this), uploads the built NPM package to an Azure Devops package feed.
```

Comment:
there's still a release pipeline?  i thought it was all just rolled up into the build pipeline now?

Reply:
Unfortunately had to keep the release pipeline. There was no way for me to publish the artifact to the existing msazure feed through the build pipeline alone.